src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.Person;
+
+import java.util.ArrayList;
+
+//an association class related Tag,Person
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.